### ISSUE TITLE
Release of validation set

### ISSUE THREAD

**Issue by ZhenghLi** [yuanqing-ai/LLM-Hierarchical-Consistency#3]
Hi, 

Could you release the validation set used for fine-tuning (Appendix D)?

Thanks.

**Comment by yuanqing-ai** [comment]
Hi, the validation set has been uploaded to `data/validation`.
